proxmox/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,sample_tags
```

Comment:
Could you fill this out with the relevant proxmox metrics that are submitted? You can do so by:
1. To obtain a list of related metrics, navigate to the [metrics summary view](https://app.datadoghq.com/metric/summary) in your sandbox and search for your metric prefix to see all related metrics.
2. Select ‘Export as CSV’.
3. Copy the information provided in the export and fill in the attached CSV with the remaining relevant information.
[metadata.csv](https://github.com/user-attachments/files/18116465/metadata.csv)